packages/dataviews/src/dataforms-layouts/row/index.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalSpacer as Spacer,
+	__experimentalVStack as VStack,
+	__experimentalHeading as Heading,
+} from '@wordpress/components';
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { FieldLayoutProps, Form, NormalizedRowLayout } from '../../types';
+import DataFormContext from '../../components/dataform-context';
+import { DataFormLayout } from '../data-form-layout';
+import { isCombinedField } from '../is-combined-field';
+import { normalizeLayout } from '../../normalize-form-fields';
+import { getFormFieldLayout } from '..';
+
+function Header( { title }: { title: string } ) {
+	return (
+		<VStack className="dataforms-layouts-row__header" spacing={ 4 }>
+			<HStack alignment="center">
+				<Heading level={ 2 } size={ 13 }>
```

Comment:
Is the heading here always at the same level and size? I can see that the Regular layout has the same implementation, so this is not a question specifically for Row.